### ISSUE TITLE
Failed to verify coredump setup for debian9

### ISSUE THREAD

**Issue by amoskong** [scylladb/scylla-artifact-tests#98]
[Test job link
](http://jenkins.cloudius-systems.com:8080/job/scylla-master-debian-deb-9-test/label=huginn/40/artifact/latest/job.log)

Current way to verify coredump setup for debian:

```
            result = process.run('sysctl kernel.core_pattern')
            assert 'scylla_save_coredump' in result.stdout
```

It works well with debian8, we might need to update it for debian9

```
2018-06-11 20:06:00,969 process          L0333 INFO | Running 'sysctl kernel.core_pattern'
2018-06-11 20:06:00,972 process          L0420 DEBUG| [stdout] kernel.core_pattern = |/usr/lib/systemd/systemd-coredump %p %u %g %s %t %e
2018-06-11 20:06:00,972 stacktrace       L0036 ERROR| 
2018-06-11 20:06:00,972 stacktrace       L0039 ERROR| Reproduced traceback from: /usr/local/lib/python2.7/dist-packages/avocado/core/test.py:436
2018-06-11 20:06:00,972 stacktrace       L0042 ERROR| Traceback (most recent call last):
2018-06-11 20:06:00,973 stacktrace       L0042 ERROR|   File "/root/avocado/tests/jenkins/slave/workspace/scylla-master-debian-deb-9-test/label/huginn/scylla-artifacts.py", line 734, in setUp
2018-06-11 20:06:00,973 stacktrace       L0042 ERROR|     self.scylla_setup()
2018-06-11 20:06:00,973 stacktrace       L0042 ERROR|   File "/root/avocado/tests/jenkins/slave/workspace/scylla-master-debian-deb-9-test/label/huginn/scylla-artifacts.py", line 720, in scylla_setup
2018-06-11 20:06:00,973 stacktrace       L0042 ERROR|     installer.run()
2018-06-11 20:06:00,973 stacktrace       L0042 ERROR|   File "/root/avocado/tests/jenkins/slave/workspace/scylla-master-debian-deb-9-test/label/huginn/scylla-artifacts.py", line 381, in run
2018-06-11 20:06:00,973 stacktrace       L0042 ERROR|     assert 'scylla_save_coredump' in result.stdout
2018-06-11 20:06:00,973 stacktrace       L0042 ERROR| AssertionError
```

**Comment by amoskong** [comment]
debian8 output:

```
2018-04-10 15:56:31,519 process          L0333 INFO | Running 'sysctl kernel.core_pattern'
2018-04-10 15:56:31,521 process          L0420 DEBUG| [stdout] kernel.core_pattern = |/usr/lib/scylla/scylla_save_coredump %e %t %p
```

**Comment by amoskong** [comment]
fixed by #99
